resources/testharness.js

```diff
@@ -3566,6 +3588,13 @@ policies and contribution forms [3].
         target[components[components.length - 1]] = object;
     }
 
+    function warn(message)
+    {
+        try {
+            console.warn(message);
```

Comment:
This will change the baselines of lots of tests in Chromium and might be painful to deal with. It's a great way to identify all the tests that need to be updated, but I think we should find all of those tests and fix them and so there's no need to have a warning.

Reply:
My intent was for this to limit the number of new tests that are introduced while we work. Your point about Chromium's infrastructure is well-taken, though. Courtesy trumps my "stopgap" idea (folks probably don't read console warnings, anyway).

As for identifying infractions, [I'm still using the synchronous-XHR-based approach](https://github.com/web-platform-tests/wpt/commit/507811a34f9d38bdc7b62b1fa49ec270d22ca46a) from the "async cleanup" days.